### PR TITLE
feat: move registerMq from common-islands to island/islet

### PR DESCRIPTION
**Pull request by kuyh**
http://git.sphd.io/support/stt-issues/issues/1537

### CODE REVIEW

File: src/islet.ts [hunk 1]
```diff
@@ -75,6 +77,29 @@ export default class Islet {
...
   }
 
+  /**
+   * Register Message Queue Adapter And Controllers
+   * @param {string} name
+   * @param {string} url
+   * @param {typeof Adapter} typeOfAdapter
+   * @param {AbstractController} controllers
+   */
+  public registerMq(name, url, typeOfAdapter, ...controllers) {
```
Comment (sshtel) [inline src/islet.ts]: why is this function public while others are protected?

Comment (sshtel) [reply src/islet.ts]: And it would be good to define type of parameters. (And ensureSampleHost too)

Comment (kuyh) [reply src/islet.ts]: cause registerAdapter is already public. so I thought registerAdapter & registerMq must be public. 
users can call those two functions from outside of islet.
ensureSampledHost and registerAdapterAndControllers are seems not to be called outside of islet

And I've added some type definitions. thanks!

Comment (sshtel) [reply src/islet.ts]: Got it!

**Pull request merged**